### ISSUE TITLE
How do I know the loading progress when loading .onnx file?

### ISSUE THREAD

**Issue by aidscooler** [huggingface/transformers.js#993]
### Question

Because the .onnx file is large(about 170M)，I decided to provide a loading progress.  Code as below: 
`      const modelSettings = {
        // Do not require config.json to be present in the repository
        config: { model_type: "custom" },
        subfolder: "",
        process_callback: (progress) => {
          modelLoadingProgress.value = Math.round(progress * 100); 
          console.log("model : " + progress)         
        }
      };
     modelSettings.device = "webgpu";
     modelSettings.dtype = "fp32";
     model = await AutoModel.from_pretrained('briaai/RMBG-1.4', modelSettings);
`
I found the process_callback never been called. Can anyone help?

**Comment by xenova** [comment]
Hi there 👋 you have a typo: it should be `progress_callback` (not proCess)

**Comment by aidscooler** [comment]
> Hi there 👋 you have a typo: it should be `progress_callback` (not proCess)

You are right,Thanks!!!  What's there in the argument progress? It shows [object object] in the console.How do I know the details of progress?

**Comment by xenova** [comment]
We also provide information like total number of bytes, so `progress` is not a single number. Feel free to `console.log` it to see the available fields. :)

**Comment by xenova** [comment]
You can now listen for `progress_total` updates when loading via pipeline or AutoModelXXX. e.g.,
```js
const progress_callback = (x) => {
  if (x.status === "progress_total") {
    console.log(x.progress); // do stuff here; you can also use x.loaded and x.total
  }
};

model = await AutoModel.from_pretrained(MODEL_ID, {
  dtype: "q4",
  device: "webgpu",
  progress_callback,
});
```
